pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -240,6 +256,69 @@ func (c *StaticResourceController) Name() string {
 	return "StaticResourceController"
 }
 
+func (c *StaticResourceController) RelatedObjects() ([]configv1.ObjectReference, error) {
+	if c.restMapper == nil {
+		return nil, errors.New("StaticResourceController.restMapper is nil")
+	}
+
+	if c.categoryExpander == nil {
+		return nil, errors.New("StaticResourceController.categoryExpander is nil")
+	}
+
+	// create lookup for resources in "all" alias
+	grs, _ := c.categoryExpander.Expand("all")
```

Comment:
@deads2k does it make sense to return an error here if `Expand` returns `false`? Or how else retry failed expansion?

Reply:
> @deads2k does it make sense to return an error here if `Expand` returns `false`? Or how else retry failed expansion?

I think the current code is ok.  It may flap briefly, but should settle quickly.  If this fails, your restmapping is likely to fail overall.

If you were trying to harden it, I would probably say that every namespaced GR is logically in "all".